components/go/pkg/api/describe.go

```diff
@@ -0,0 +1,68 @@
+package api
+
+import (
+	"strings"
+
+	config "github.com/chef/chef-licensing/components/go/pkg/config"
+)
+
+type LicenseDetail struct {
+	LicenseKey   string  `json:"licenseKey"`
+	SerialNumber string  `json:"serialNumber"`
+	LicenseType  string  `json:"licenseType"`
+	Name         string  `json:"name"`
+	Start        string  `json:"start"`
+	End          string  `json:"end"`
+	Status       string  `json:"status"`
+	Limits       []Limit `json:"limits"`
+}
+
+type Limit struct {
+	Software string `json:"software"`
+	ID       string `json:"id"`
+	Amount   int    `json:"amount"`
+	Measure  string `json:"measure"`
+	Used     int    `json:"used"`
+	Status   string `json:"status"`
+}
+
+type Asset struct {
+	ID       string `json:"id"`
+	Name     string `json:"name"`
+	Entitled bool   `json:"entitled"`
+	From     []struct {
+		LicenseKey string `json:"license"`
+		Status     string `json:"status"`
+	} `json:"from"`
+}
+
+type LicenseDescribe struct {
```

Comment:
JSON key is not consistent. Fewplaces it is start with smallcase `license` and few places start with capital `Software Features`. 
If it is internal JSON then we can fix it and make it consistent else no need

Reply:
These are coming from the server.